docs/interactions/Slash_Commands.md

```diff
@@ -666,7 +666,7 @@ headers = {
 r = requests.put(url, headers=headers, json=json)
 ```
 
-## Endpoints
+### Endpoints
```

Comment:
Does this break links in other places? IIRC `##` can be a standalone link like `INTERACTIONS_SLASH_COMMANDS/endpoints` whereas `###` nests it like `INTERACTIONS_SLASH_COMMANDS/something-endpoints`.

Just double check

Reply:
This seems to only apply to 4 # or more, not 3.
![image](https://user-images.githubusercontent.com/43933794/123168030-09f3d380-d43d-11eb-9bd2-5e1fddfe84cd.png)
[`/channel-object` here is ###.](https://discord.com/developers/docs/resources/channel#channel-object)